src/screens/TopicMemberScreen/index.js

```diff
@@ -228,13 +180,17 @@ const TopicMemberScreen = () => {
         onCancelToken={onTokenCancel}
         placeholderText={StringConstant.topicMemberPlaceholder}
         setIsFirstTimeOpen={setIsFirstTimeOpen}
-        hideBackIcon={true}
+        getSearchLayout={setSearchHeight}
       />
       <ScrollView
+        decelerationRate="fast"
+        scrollEventThrottle={1}
         style={styles.fragmentContainer}
-        contentContainerStyle={styles.fragmentContentContainer}
+        contentContainerStyle={[styles.fragmentContentContainer, {paddingTop: headerShowHeight}]}
         keyboardShouldPersistTaps="handled"
-        onScroll={handleScrollEvent}>
+        keyboardDismissMode="on-drag"
+        removeClippedSubviews={true}
+        onScroll={handleScroll}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ScrollView` component has been updated with new props like `decelerationRate`, `scrollEventThrottle`, `keyboardDismissMode`, and `removeClippedSubviews`. These changes could improve the performance of the scroll view. The `onScroll` prop has been updated to use the new `handleScroll` function.
```javascript
<ScrollView
  decelerationRate="fast"
  scrollEventThrottle={1}
  style={styles.fragmentContainer}
  contentContainerStyle={[styles.fragmentContentContainer, {paddingTop: headerShowHeight}]}
  keyboardShouldPersistTaps="handled"
  keyboardDismissMode="on-drag"
  removeClippedSubviews={true}
  onScroll={handleScroll}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->